raiden/transfer/mediated_transfer/events.py

```diff
@@ -11,7 +11,7 @@
 # channel_identifier can never be 0. We make this a requirement in the client and use this fact
 # to signify that a channel_identifier of `0` passed to the messages adds them to the
 # global queue
-CHANNEL_IDENTIFIER_GLOBAL_QUEUE: typing.ChannelID = 0
+CHANNEL_IDENTIFIER_GLOBAL_QUEUE: typing.ChannelID = typing.ChannelID(0)
```

Comment:
do you need to specify the type twice here? If you assign `typing.ChannelID` to the variable I think it is understood to be of that type.